prow/errorutil/aggregate.go

```diff
@@ -18,21 +18,23 @@ package errorutil
 
 import (
 	"fmt"
+	"strings"
```

Comment:
This used to be a copy paste of utils in `k/k` I couldn't figure out how to import... I don't think they're in any staging repo. Anyway, I don't think it matters if we fork this but just FYI